### DISCUSSION TITLE
How do I increase the amount of timeout before suggestions show/update?

### DISCUSSION THREAD

**Discussion by SephReed** [zed-industries/zed#17260] [Feature Requests]
Is it possible to increase the delay between each input and the moment it starts making suggestions for what comes next?

https://github.com/user-attachments/assets/7ebb3ff3-88b6-4ebc-a9c2-c52a20ded279

I would like to set the delay to 300-500 ms so I don't have a little box partying it's butt off atop my cursor for every keystroke.  I'm happy for all the energy it has (impressively responsive), but I'm personally trying to relax and enter a flow state.

**Comment by SephReed** [comment]
In general, are there any settings pertaining to the interactions that lead to pop-ups?  Having come from the days of coding in black and white raw text editors, this is simply more information than I need in order to write "forEach."

<img width="1104" alt="Screenshot 2024-09-02 at 5 09 42 PM" src="https://github.com/user-attachments/assets/d7dbf381-b504-40f3-8450-139d26e3676a">

Further, it's covering up the code I just wrote which makes it much harder to remember what I was doing:

<img width="703" alt="image" src="https://github.com/user-attachments/assets/4fc3284f-b51d-43fb-863e-a95c9bae125a">
<img width="502" alt="image" src="https://github.com/user-attachments/assets/7df672f4-d96d-4597-a82b-f7b7413e7290">

**Comment by SephReed** [comment]
There is some stuff!

<img width="797" alt="image" src="https://github.com/user-attachments/assets/744ec229-57a3-47fa-91fa-0a19f277c425">

These too:
```
"auto_signature_help": false,
  "show_signature_help_after_edits": false,
  ```

**Comment by SephReed** [comment]
- https://github.com/zed-industries/zed/issues/17294

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
